crypto/include/internal/md32_common.h

```diff
@@ -93,6 +93,12 @@
 # error "HASH_BLOCK_DATA_ORDER must be defined!"
 #endif
 
+typedef unsigned int ossl_uint
+#ifdef __GNUC__
+  __attribute((__aligned__(1)))
+#endif
+;
```

Comment:
I can't say I'm a great fan of this construct which results in a single `;` on a line on its own.

Reply:
you did not see the earlier attempts :smiley:

Reply:
How about just:

````
#ifdef __GNUC__
typedef unsigned int ossl_uint __attribute((__aligned__(1)));
#else
typedef unsigned int ossl_uint;
#endif
````

Reply:
Yes, that was my second thought as well.  Thanks!